README.md

```diff
@@ -22,7 +22,7 @@ More information on the REST interface of ecChronos is described in [REST.md](do
 
 ### Prerequisites
 
-* JDK8
+* JDK11
```

Comment:
CONTRIBUTING.md also has a reference to prerequisites that specify JDK8.
Prerequisites

    Maven
    JDK8
    Docker (for test setup)
    Python